### ISSUE TITLE
'ToS;DR' should be labeled as 'Terms of Service; Didn't Read' 📖

### ISSUE THREAD

**Issue by jagtalon** [duckduckgo/duckduckgo-privacy-extension#260]
'ToS;DR' should be labeled as 'Terms of Service; Didn't Read' in `aria-label` so that screen readers don't ry to pronounce this weird string.

<img width="1680" alt="" src="https://user-images.githubusercontent.com/81969/43592047-b6762f2c-9642-11e8-9e0f-e4e60d4276bf.png">
